app/src/sharedTest/java/org/oppia/app/options/OptionsFragmentTest.kt

```diff
@@ -0,0 +1,297 @@
+package org.oppia.app.options
+
+import android.app.Application
+import android.content.Context
+import android.content.res.Resources
+import android.widget.SeekBar
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.CoordinatesProvider
+import androidx.test.espresso.action.GeneralClickAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Tap
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.domain.profile.ProfileManagementController
+import org.oppia.domain.profile.ProfileTestHelper
+import org.oppia.util.logging.EnableConsoleLog
+import org.oppia.util.logging.EnableFileLog
+import org.oppia.util.logging.GlobalLogLevel
+import org.oppia.util.logging.LogLevel
+import org.oppia.util.threading.BackgroundDispatcher
+import org.oppia.util.threading.BlockingDispatcher
+import javax.inject.Inject
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+/** Tests for [OptionsFragment]. */
+@RunWith(AndroidJUnit4::class)
+class OptionsFragmentTest {
+
+
+  @Inject lateinit var profileTestHelper: ProfileTestHelper
+  @Inject lateinit var profileManagementController: ProfileManagementController
+  @Inject lateinit var context: Context
+
+  @Before
+  @ExperimentalCoroutinesApi
+  fun setUp() {
+    Intents.init()
+    setUpTestApplicationComponent()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private fun setUpTestApplicationComponent() {
+    DaggerOptionsFragmentTest_TestApplicationComponent.builder()
+      .setApplication(ApplicationProvider.getApplicationContext())
+      .build()
+      .inject(this)
+  }
+
+  @get:Rule
+  var optionActivityTestRule: ActivityTestRule<OptionsActivity> = ActivityTestRule(
+    OptionsActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Test
+  fun testOptionFragment_clickStoryTextSize_changeTextSizeToLargeSuccessfully() {
```

Comment:
This test cases needs to verify in the end that the text-size has been changed to large.

Reply:
This PR is split into 2 and Test cases will be added in part 2 of this PR #614.